### ISSUE TITLE
Light mode tooltips use dark mode theme

### ISSUE THREAD

**Issue by lancegliser** [themesberg/flowbite-react#1438]
- [X] I have searched the [Issues](https://github.com/themesberg/flowbite-react/issues) to see if this bug has already been reported
- [X] I have tested the latest version

## Steps to reproduce

1. Go to https://flowbite-react.com/docs/components/tooltip#trigger-type
2. Click on "Tooltip click"
3. Review classes

## Current behavior

It appears there is a theme mismatch for tooltips in light mode.
Image from: https://flowbite-react.com/docs/components/tooltip#theme

<img width="860" alt="image" src="https://github.com/themesberg/flowbite-react/assets/12085479/d4eeca84-647b-455b-a4da-9f64f5b80d06">

`style.light` should be: `border border-gray-200 bg-white text-gray-900`.
`style.auto` should be: `border border-gray-200 bg-white text-gray-900 dark:border-none dark:bg-gray-700 dark:text-white`

The inspector seems to indicate that we're getting `style.dark`:

<img width="320" alt="image" src="https://github.com/themesberg/flowbite-react/assets/12085479/8963e209-6207-4cb1-aa7e-d55e2cc5b9e7">

`style.dark` is `bg-gray-900 text-white dark:bg-gray-700`

## Expected behavior

White background tooltips with dark text

**Comment by SutuSebastian** [comment]
I think there's a misunderstanding here, when we're talking about `light`, `dark` and `auto` tooltip we're talking about the style of it, not if we're in `light` or `dark` mode in tailwind.

Here's a screenshot of the `flowbite` [Tooltip](https://flowbite.com/docs/components/tooltips/#tooltip-styles)

<img width="1443" alt="Screenshot 2024-06-27 at 11 42 36" src="https://github.com/themesberg/flowbite-react/assets/41998826/99a82c40-7e14-4062-9e9f-3ce08b836021">

Now here's some screenshots of the `flowbite-react` [Tooltip](https://flowbite-react.com/docs/components/tooltip#tooltip-styles)

<img width="1579" alt="Screenshot 2024-06-27 at 11 43 22" src="https://github.com/themesberg/flowbite-react/assets/41998826/c32c0ce2-eaff-4235-be96-5fb1e31b7fcf">

Notice the `style` prop

<img width="898" alt="Screenshot 2024-06-27 at 11 43 37" src="https://github.com/themesberg/flowbite-react/assets/41998826/e5110d9e-cd80-4e2e-9f40-4966e83f8561">

**Comment by lancegliser** [comment]
I believe I did miss that. Apologies, it even is documented. Though, a couple concerns about even that:

The docs for that section state:
> Use the style prop to change the style of the tooltip. The default style is light and you can also use dark.

That does not appear to be the default however:
<img width="855" alt="image" src="https://github.com/themesberg/flowbite-react/assets/12085479/1e291466-6206-4af0-b5d7-a53cc269f87d">

I checked, and my `flowbite-theme-mode` is set to `light`. 

I also wonder if the right default might be `auto`, though that's different than a bug.
